sdk/include/opentelemetry/sdk/trace/recordable.h

```diff
@@ -146,6 +147,25 @@ class Recordable
    */
   virtual void SetInstrumentationLibrary(
       const InstrumentationLibrary &instrumentation_library) noexcept = 0;
+
+  /**
+   * Set the span limits parameters for the span.
+   * @param span_limits span limits struct to be set
+   */
+  void SetSpanLimit(const SpanLimits &span_limits)
+  {
+    span_limit_ = span_limits;
+  }
+
+protected:
+
+  SpanLimits GetSpanLimits()
+  {
+    return span_limit_;
+  }
+
+private:
+  SpanLimits span_limit_;  
```

Comment:
I don't think we should add `SpanLimits` as a data member here. It will get allocated every time a new recordable is created ( i.e, every time a new span is created ). And SpanLimits values are not going to change.

Reply:
This is not intended change, maybe didn't sync correctly.